community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/resume.py

```diff
@@ -132,9 +118,28 @@ def instance_properties(nodeset:object, model:str, placement_group:Optional[str]
 
     # Override with properties explicit specified in the nodeset
     props.update(nodeset.get("instance_properties") or {})
-    
     return props
 
+def update_reservation_props(reservation:object, props:object) -> None:
+    props.reservationAffinity = {
+        "consumeReservationType": "SPECIFIC_RESERVATION",
+        "key": f"compute.{util.universe_domain()}/reservation-name",
+        "values": [reservation.bulk_insert_name],
+    }
+
+    if reservation.dense:
```

Comment:
It should be `if reservation.dense OR reservation_comes_from_DWS_calendar`

Reply:
Added but going to sync offline about some questions I have with this.